api-template-java/src/test/java/com/slf/services/HandlerTest.java

```diff
@@ -17,23 +17,116 @@
 package com.slf.services;
 
 import java.util.HashMap;
+import java.util.Map;
```

Comment:
This is our template file that gets committed to the user repo when they create a Java service, so it wouldn't make sense to add unit tests that would fail once the user starts changing code.